lib/tasks/scripts.js

```diff
@@ -47,30 +65,27 @@ module.exports = function(gulp, config) {
 							}
 						}
 					}, {
+						// ES6+
 						test: function(filepath) {
-							var ext = path.extname(filepath)
+							const ext = path.extname(filepath)
 							return jsExtensions.indexOf(ext) !== -1
 						},
-						use: {
-							loader: require.resolve('babel-loader'),
-							options: {
-								presets: [
-									[ require.resolve('@babel/preset-env'), { modules: false } ],
-									require.resolve('@babel/preset-react'),
-									require.resolve('@babel/preset-typescript'),
-								],
-								plugins: [
-									'@babel/plugin-syntax-decorators',
-									'@babel/plugin-proposal-decorators',
-									'@babel/plugin-syntax-class-properties',
-									'@babel/plugin-proposal-class-properties',
-									'@babel/plugin-syntax-object-rest-spread',
-									'@babel/plugin-proposal-object-rest-spread',
-								].map(require.resolve),
-								cacheDirectory: true,
-							}
-						}
-					},
+						use: [
+							{ loader: require.resolve('cache-loader') },
+							{ loader: require.resolve('babel-loader'), options: babelOptions },
+						]
+					}, {
+						// TypeScript
+						test: function(filepath) {
```

Comment:
Když už to měníš, tahle funkce mohla být něco jako `getExtensionTester(tsExtensions)`, což by vrátilo něco jako tohle ‒ o pár řádků výše se opakuje takřka totéž. Ale je to jen kosmetická drobnost.